docs/src/getting-started/overview.md

```diff
@@ -0,0 +1 @@
+# Deploy a Sway contract
```

Comment:
Remove empty files like this (there are a few), along with any links to them in ToCs.